docs/performance/database-indexes.rst

```diff
@@ -0,0 +1,19 @@
+Database indexes
+================
+
+Indexing tables allows you to decrease fetching time from database.
+
+As an example lets take a look on customers list
+
+Default index page is sorted by registration date, to create table index all you need to do is modify `src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Customer.orm.xml` and add
```

Comment:
The problem with this line is that developers reading this doc will not be able to edit the `src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Customer.orm.xml` file - as it will be part of vendor code. This change should be done with annotations here: https://github.com/Sylius/Sylius-Standard/blob/1.10/src/Entity/Customer/Customer.php#L12

Can we also link to doctrine docs? https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/annotations-reference.html#annref_haslifecyclecallbacks